src/scales/timeScale.ts

```diff
@@ -38,11 +39,7 @@ export module Scales {
     }
 
     protected _defaultExtent(): Date[] {
-      let now = new Date();
-      let endTimeValue = now.valueOf();
-      now.setDate(now.getDate() - 1);
-      let startTimeValue = now.valueOf();
-      return [new Date(startTimeValue), new Date(endTimeValue)];
+      return [new Date("1970-01-01"), new Date("1970-01-02")];
```

Comment:
Not a big deal, but I personally like the `(year, month, day)` constructor since I find it to be less error prone.